src/DataModel/IClass.ts

```diff
@@ -0,0 +1,5 @@
+import {IResource} from "./IResource";
+
+export interface IClass extends IResource
```

Comment:
On the wire - indeed. But I think client's model should be rooted in hydra's model somehow.